app/src/androidTest/java/com/github/houseorganizer/houseorganizer/HouseSelectionActivityTest.java

```diff
@@ -36,6 +37,9 @@ public class HouseSelectionActivityTest {
     private static FirebaseFirestore db;
     private static FirebaseAuth auth;
 
+    @Rule
+    public GrantPermissionRule permissionRules = GrantPermissionRule.grant(android.Manifest.permission.ACCESS_FINE_LOCATION);
```

Comment:
So we have to put the rule of the premission in the tests class concerned ?

Reply:
That's a good question; the offline screen also uses some sort of permission

Reply:
I think so, but I haven't still been able to prevent the pop up from appearing